server/apps/archive/common.py

```diff
@@ -184,7 +184,7 @@ def remove_unwanted(doc):
     """
 
     # _type attribute comes when queried against Elastic and desk comes while fetching an item from ingest
-    for attr in ['_type', 'desk']:
+    for attr in ['_type', 'desk', 'archived']:
         if attr in doc:
```

Comment:
I want to avoid the existing 'wrong' data in content in master-sd being proliferated by duplication. After all the already fetched data with wrong value disappear we can revisit this.